backend/services/reaction.py

```
@@ -0,0 +1,77 @@
+from sqlalchemy.orm import Session
+from models.post import Like, Comment, Post
+from models.user import User
+from schemas.postReaction import LikeCreate
+from datetime import datetime
+from models.user import User
```

Ruff (0.8.2)

Redefinition of unused `User` from line 3

Remove definition: `User`

(F811)